reV/qa_qc/summary.py

```diff
@@ -191,9 +191,11 @@ def summarize_dset(self, ds_name, process_size=None, max_workers=None,
             ds_shape, _, ds_chunks = f.get_dset_properties(ds_name)
             if len(ds_shape) > 1:
                 sites = np.arange(ds_shape[1])
-                if max_workers > 1:
+                if max_workers != 1:
+                    if process_size is None and ds_chunks is not None:
```

Comment:
@MRossol, i think this is what you were trying to do? The code ran into some issues i think when ds_chunks was none.